aws/resource_aws_route_test.go

```diff
@@ -737,6 +761,10 @@ resource "aws_vpc" "foo" {
 
 resource "aws_internet_gateway" "foo" {
   vpc_id = "${aws_vpc.foo.id}"
+
+  tags {
+    Name = "terraform-testacc-route-with-vpc-endpoint"
+  }
 }
 
 resource "aws_route_table" "foo" {
```

Comment:
_minor_ it'd be nice to make the spacing consistent in this file - but it's by no means a blocker

Reply:
Hah yeah. You're absolutely right. 😅  I need to create a technical debt issue to clean these up across the entire repository... its very inconsistent at the moment. We're preferring 2 space indentations in the test configurations. Actually why not, here we go! #3576